.github/workflows/create-sdk-update-pr.yml

```diff
@@ -19,7 +19,7 @@ jobs:
   update:
     runs-on: ubuntu-latest
     env:
-      GH_TOKEN: ${{ github.token }}
+      GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
How does this trigger the CI due to the change?

Reply:
The "build in" GH token never triggers other workflows. (To avoid accidental workflow trigger loops)

The new token is a personal access token, and as such will trigger all the workflows a regular user would trigger :)

Reply:
TIL! Thanks for the context!